megalodon/tsconfig.mjs.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "ES2020",
```

Comment:
Is there any reason that ES2020 is being used instead of ES2023 or ESNext?

Reply:
There is no reason. Is ES2023 better?

Reply:
Yes, it's generally best to keep the ES module version either as the current year or `ESNext`